test-utils/src/spl.rs

```diff
@@ -59,7 +59,7 @@ pub async fn mint_tokens_helper<const INDEXED_ARRAY_SIZE: usize, R: RpcConnectio
         spl_token::state::Account::unpack(&rpc.get_account(pool).await.unwrap().unwrap().data)
             .unwrap()
             .amount;
-    let event = rpc
+    let (event, _signature) = rpc
```

Comment:
```suggestion
    let (event, _) = rpc
```
this works as well and produces less noise.

Reply:
agreed just saying